templates/guide/stage.html

```diff
@@ -0,0 +1,46 @@
+{% extends "_base.html" %}
+
+{% block css %}
+<link rel="stylesheet" href="/static/css/forms.css">
+<style>
+.delete-button {
+  margin-left: 20px;
+}
+</style>
+{% endblock %}
+
+{% block subheader %}
+<div id="subheader">
+  <h2>Edit feature: {{ feature.name }}</h2>
+  {% if user.is_admin %}
+    <button class="delete-button" data-id="{{feature_id}}">Delete</button>
+  {% endif %}
+</div>
+{% endblock %}
+
+{% block content %}
+<section>
+  <form name="feature_form" method="POST" action="{{current_path}}">
+    <table>
+      {{ feature_form }}
+    </table>
+    <ul>
+      <li>
+        <input type="checkbox" name="intent_to_implement" id="id_intent_to_implement">
+        <label for="id_intent_to_implement">Generate "Intent to <span id="id_intent_stage_name"></span>" email on next page (you will copy &amp; paste it to send)</label>
```

Comment:
Seems not used? Can it be removed?

Reply:
Yes, but I didn't want to get into changing that part in this CL.  I added a TODO comment to redo that part and I will follow up with a separate CL.